tests/drivers/pci_enum/testcase.yaml

```diff
@@ -1,5 +1,4 @@
 tests:
 -   test:
-        filter: CONFIG_PCI
```

Comment:
Right, but ultimate goal is to stop using filter for this type of thing. It is slow, it is also not consistent, here Qemu does not have CONFIG_PCI set by default and there is a hack in the Makefile to force it and I just noticed the test is not even running by default in Qemu. I already added support for HW feature matching, so I will be using it here for the first time, one more patch coming on top of this....